### DISCUSSION TITLE
parallel GP with example <test_GP_LJ_fresh.yaml>

### DISCUSSION THREAD

**Discussion by TSMtang** [mir-group/flare#345] [Q&A]
Dear Flare developer,

I've been using flare SGP and it's excellent, but when I try to use the full GP, I find that it's not doing parallel calculation. 

Here is what I did:
1) use example file <test_GP_LJ_fresh.yaml> as template
2) modify "n_cpus: 1" to "n_cpus: 16"
3)  export OMP_NUM_THREADS=2
4) use command $ flare-otf input.yaml

In the flare output file (.out) it knows how many cores I'm giving, but when I check my system monitor, only two CPUs are working. And no matter how I modify that "n_cpus" and "OMP_NUM_THREADS", it's not doing parallel calculation.

Could you please give me a hint of what might be the problem? I'm confused because SGP has no problem, whereas GP has this issue.

Best regards
Shengming Tang

**Comment by YuuuXie** [comment]
Hi @TSMtang, sorry for the very delayed reply. I never receive notifications from our GitHub Discussions so I did not notice there was a question... Are you still facing the problem? Based on the documentation, it seems that there is also the arg `parallel` that needs to be `True`:
https://github.com/mir-group/flare/blob/960757e5802fe60eb187a43dba19f763a1c6e1e3/flare/bffs/gp/gp.py#L66

**Reply by YuuuXie** [reply]
P.S. Just modified my notification settings so I won't miss anything... Did not realize when did I "unwatch"ed our repo mistakenly
